activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -45,130 +45,780 @@ def primary_key(name, type = :primary_key, **options)
         ##
         # :method: bigserial
         # :call-seq: bigserial(*names, **options)
+        #
+        # Adds a +bigserial+ column, which is a PostgreSQL auto-incrementing 64-bit integer.
+        #
+        # This is commonly used for primary keys or large counters that require a greater range
+        # than +serial+ (which is 32-bit).
+        #
+        # Example:
+        #
+        #   t.bigserial :event_id
+        #
+        # You can also explicitly declare it as the primary key:
```

Comment:
For the API documentation we [prefer to avoid "you"](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns).
I'm not sure about the reasoning, but I think it's common for manuals.
Probably because it uses a more formal tone instead of a conversational one.
Maybe something like:
```suggestion
        # The column can also be explicitly declared as the primary key:
```
I've made similar suggestions further on.